src/pages/log-match/components/userLookup.tsx

```diff
@@ -0,0 +1,67 @@
+import clsx from "clsx";
+import { SearchIcon } from "../../../lib/icons";
+
+interface Props extends JSX.HtmlInputTag {
+  input: string;
+  label: string;
+  needed?: boolean;
+}
+
+export const UserLookUp = ({ ...props }: Props) => (
+  <>
+    <script>
+      {function checkUserKeydown(event: Event) {
+        // Don't submit a search on enter or when selecting an entry with the mouse
+        return event instanceof KeyboardEvent && event.key !== "Enter";
+      }}
+    </script>
+    <SearchIcon />
+    <input
+      id={`${props.input}-input`}
+      form="matchForm"
+      hx-trigger="keyup[checkUserKeydown.call(this, event)] changed delay:300ms"
+      hx-sync="this:replace"
+      hx-swap="innerHtml"
+      hx-get="/match/search"
+      hx-indicator=".progress-bar"
+      hx-target={`#${props.input}-results`}
+      hx-params="name"
+      name="name"
+      placeholder=" "
+      autocomplete="off"
+      class={clsx([
+        "peer block w-full appearance-none px-0 py-2.5 pl-10 text-sm text-white",
+        "border-0 border-b-2 border-gray-300 bg-transparent",
+        "focus:border-blue-500 focus:outline-none focus:ring-0",
+      ])}
+      {...props}
+      _="on focus remove @hidden from next <div/>"
+    />
+    <label
+      for={`${props.input}-input`}
+      class={clsx([
```

Comment:
Sometimes it's done as `clsx(["", "",""])` other times it's done as `clsx([""], [""], [""])` is there a reason for that or could it be done consistently?

Reply:
Just me not being consistent - I will use this format `clsx(["", "",""])`. Good catch!